src/kirby/components/toast/config/toast-config.ts

```diff
@@ -0,0 +1,8 @@
+
+export interface ToastConfig {
+  message: string;
+  messageType?: MessageType;
```

Comment:
Shouldn't be optional
```suggestion
  messageType: MessageType;
```